src/dgNonlinearForm.cpp

```diff
@@ -355,8 +355,8 @@ void DGNonLinearForm::evalFaceFlux_gpu() {
                          d_normal + offset + k * dim,
                          Rflux);
 
-      d_flux->ComputeViscousFluxes(u1, gradUp1, xyz[0], vFlux1);
-      d_flux->ComputeViscousFluxes(u2, gradUp2, xyz[0], vFlux2);
+      d_flux->ComputeViscousFluxes(u1, gradUp1, xyz[0], 0.0, vFlux1);
```

Comment:
Clearly doesn't work yet on the gpu.  Need to add a check that exits if user tries to run with SGS on the gpu.

Reply:
Done in b749506